Integrations/ArcSightESM/ArcSightESM.py

```diff
@@ -639,11 +678,12 @@ def get_entries_command():
             'Type': entryTypes['note'],
             'ContentsFormat': formats['json'],
             'Contents': entry_context,
-            'HumanReadable': tableToMarkdown(name='Active List entries', t=filtered, removeNull=True),
+            'HumanReadable': tableToMarkdown(name='Active List entries: {}'.format(resource_id), t=filtered,
```

Comment:
replace all demisto.results with `return_outputs`

Reply:
I had a warper function for `demisto.results` similar to `return_outputs`, and you told me to remove it as it's easier to debug when you see all outputs in a straight forward way like here.